packages/asset-service/src/generateAssetData/osmosis/index.ts

```diff
@@ -17,33 +18,72 @@ type OsmoAsset = {
   name: string
   display: string
   symbol: string
+  traces: {
+    type: string
+    counterparty: {
+      chain_name: string
+      bse_denom: string
+      channel_id: string
+    }
+    chain: {
+      channel_id: string
+    }
+  }[]
   logo_URIs: {
     png?: string
     svg?: string
   }
   coingecko_id: string
-  pools?: {
-    [key: string]: number
-  }
+  keywords: string[]
 }
 
 type OsmosisAssetList = {
-  chain_id: string
+  chain_name: string
   assets: OsmoAsset[]
 }
 
 export const getAssets = async (): Promise<Asset[]> => {
+  /** Helper function to get latest commit hash for default branch of given repo.
+   * This is used to construct githack URLs */
+  const getCurrentHash = memoize(async (repository: string): Promise<string> => {
+    /* Trim any trailing slashes */
+    while (repository.charAt(repository.length - 1) === '/') {
+      repository = repository.slice(0, -1)
+    }
+
+    const {
+      data: { default_branch: defaultBranch },
+    } = await axios.get(`https://api.github.com/repos/${repository}`)
```

Comment:
Also note in case we keep this function that this will fire XHRs to GitHub API unauthenticated. Not an issue here given the fact that we only run `generateAssetData` so few times, but generally a better idea to assume `gh` is installed and spawn a gh process instead